### DISCUSSION TITLE
Questions about using blueman in cent os

### DISCUSSION THREAD

**Discussion by CollinKa** [blueman-project/blueman#1779] [Q&A]
Hi, I am working on a Bluetooth project which requires me to do the serial communication with a lab pc with cent os7. However, blueman doesn’t support cent os7. Is there any solution to make it work on cent os?

**Comment by cschramm** [comment]
What's the problem with blueman on Cent OS 7? Unsatisfied dependencies?

**Comment by CollinKa** [comment]
Is it possible to make it run under python 3.7? If so what should I do?

Thanks
Haoliang

Christopher Schramm ***@***.***>于2022年7月10日 周日下午12:38写道：

**Reply by infirit** [reply]
Use blueman 2.2.5 which was released a few weeks ago.

**Reply by cschramm** [reply]
2.3 supports Python 3.7 as well. Only the main branch dropped support very recently.
